src/tests/unit-tests/signal-processing-blocks/acquisition/beidou_b2ad_pcps_acquisition_test.cc

```diff
@@ -0,0 +1,264 @@
+/*!
+ * \file beidou_b2ad_pcps_acquisition_test.cc
+ * \brief  Tests a PCPS acquisition block for Glonass L1 C/A signals
+ * \author Gabriel Araujo, 2017. gabriel.araujo.5000(at)gmail.com
```

Comment:
This is a new file, and even though you copy and paste it you should be the sole author